### PR TITLE
Add full screen mode

### PR DESCRIPTION
**Pull request by crubier**
# Feature

Add a full screen button, quick win.

Screenshot 2021-09-22 at 10 39 03

### CODE REVIEW

File: typescript/web/src/components/labelling-tool/view-tool-bar/full-screen-tool.tsx [hunk 1]
```diff
@@ -0,0 +1,127 @@
...
+/**
+ * @return {boolean} Fullscreen is supported by the current platform.
+ */
+function isFullScreenSupported() {
+  if (!isInWindowScope) {
+    // Assume it is supported for most users.
+    return true;
+  }
+  const { body } = document;
+  return !!(
```
Comment (nacho-carnicero) [inline typescript/web/src/components/labelling-tool/view-tool-bar/full-screen-tool.tsx:27]: Why a double negation?

File: typescript/web/src/components/labelling-tool/view-tool-bar/full-screen-tool.tsx [hunk 2]
```diff
@@ -0,0 +1,127 @@
...
+// /**
+//  * @return {boolean} Element is currently in fullscreen.
+//  */
+// function isFullScreen() {
+//   return !!(
+//     document.webkitIsFullScreen ||
+//     document.msFullscreenElement ||
+//     document.fullscreenElement
+//   );
+// }
```
Comment (nacho-carnicero) [inline typescript/web/src/components/labelling-tool/view-tool-bar/full-screen-tool.tsx:43]: Remove if not needed

File: typescript/web/src/components/labelling-tool/view-tool-bar/full-screen-tool.tsx [hunk 3]
```diff
@@ -0,0 +1,127 @@
...
+ * Request to fullscreen an element with keyboard input.
+ * @param {HTMLElement} element Element to request fullscreen
+ */
+function requestFullScreenWithKeys(element: HTMLElement) {
+  if (element.webkitRequestFullscreen) {
+    element.webkitRequestFullscreen();
+  } else {
+    requestFullScreen(element);
+  }
+}
```
Comment (nacho-carnicero) [inline typescript/web/src/components/labelling-tool/view-tool-bar/full-screen-tool.tsx:69]: There's no keyboard shortcut for enabling fullscreen right? Plus `element.webkitRequestFullscreen()` is already called in `requestFullScreen`, is this really needed?

Comment (crubier) [reply typescript/web/src/components/labelling-tool/view-tool-bar/full-screen-tool.tsx:69]: All these functions are just a copy paste from openlayers code as said in the Comment on top. I can customize it if needed but I liked the idea of just keeping it as is

File: typescript/web/src/components/labelling-tool/view-tool-bar/full-screen-tool.tsx (hunk 1 above)
Comment (crubier) [reply typescript/web/src/components/labelling-tool/view-tool-bar/full-screen-tool.tsx:27]: I copied this code from openlayers. It allows to cast to boolean: ` !!null === false`   and `!!{} === true`

File: typescript/web/src/components/labelling-tool/view-tool-bar/full-screen-tool.tsx (hunk 3 above)
Comment (crubier) [reply typescript/web/src/components/labelling-tool/view-tool-bar/full-screen-tool.tsx:69]: But yes I can add a keyboard shortcut

**Pull request merged**